upgraderesponder/service.go

```diff
@@ -216,6 +228,82 @@ func (s *Server) createContinuousQueries(dbName string) error {
 	return nil
 }
 
+func (s *Server) addContinuousQuerieStringsFromFields(databaseName string, queryStrings map[string]string) error {
+	influxQuery := influxcli.NewQuery(fmt.Sprintf("SHOW FIELD KEYS FROM %s", InfluxDBMeasurement), databaseName, "")
+	influxResp, err := s.influxClient.Query(influxQuery)
+	if err != nil {
+		return errors.Wrapf(err, "failed to get all field keys from %s", InfluxDBMeasurement)
+	}
+
+	keyDataTypeMap := s.getKeyDataTypeMapFromQueryResponse(*influxResp, nil)
+	for keyCamel, dataType := range keyDataTypeMap {
+		key := utils.ToSnakeCase(keyCamel)
+		continuousQuery := fmt.Sprintf(InfluxDBContinuousQueryFmt, key)
+
+		var measurement string
+		switch dataType {
+		case "float":
+			measurement = fmt.Sprintf(InfluxDBMeasurementMeanFmt, key)
+			queryStrings[continuousQuery] = fmt.Sprintf("CREATE CONTINUOUS QUERY %v ON %v BEGIN SELECT MEAN(%v) AS total INTO %v FROM %v GROUP BY time(%v) END",
+				continuousQuery, databaseName, key, measurement, InfluxDBMeasurement, InfluxDBContinuousQueryPeriod)
+		case "boolean":
+			measurement = fmt.Sprintf(InfluxDBMeasurementCountFmt, key)
+			queryStrings[continuousQuery] = fmt.Sprintf("CREATE CONTINUOUS QUERY %v ON %v BEGIN SELECT COUNT(%v) AS total INTO %v FROM %v WHERE %v = true GROUP BY time(%v) END",
+				continuousQuery, databaseName, key, measurement, InfluxDBMeasurement, key, InfluxDBContinuousQueryPeriod)
+		default:
+			measurement = fmt.Sprintf(InfluxDBMeasurementCountFmt, key)
+			queryStrings[continuousQuery] = fmt.Sprintf("CREATE CONTINUOUS QUERY %v ON %v BEGIN SELECT COUNT(%v) AS total INTO %v FROM %v GROUP BY time(%v) END",
+				continuousQuery, databaseName, key, measurement, InfluxDBMeasurement, InfluxDBContinuousQueryPeriod)
+		}
+	}
+	return nil
+}
+
+func (s *Server) addContinuousQuerieStringsFromTags(databaseName string, queryStrings map[string]string) error {
```

Comment:
I am thinking that the main focus for the extra tag info and extra field info is to understand the distribution of different values/settings currently. Therefore, I think we don't need to create continuous queries because we can use the data in the last hour to visualize all kind of distribution. What do you think?